modules/src/application/ics20_fungible_token_transfer/error.rs

```diff
@@ -23,6 +23,9 @@ pub enum Kind {
         "Destination channel not found in the counterparty of port_id {0} and channel_id {1} "
     )]
     DestinationChannelNotFound(PortId, ChannelId),
+
+    #[error("Invalid packet timeout timestamp value")]
```

Comment:
Could you please change this message and other ones above to start with a lowercase letter?